modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -224,138 +223,130 @@ xref:back-up-bonita-bpm-platform.adoc[Back up your platform] and database in cas
+ ** Perform a diff between the source version and the target version of `tenants/[TENANT_ID]/conf/compound-permissions-mapping.properties` and put the additional lines into the file `tenants/[TENANT_ID]/conf/compound-permissions-mapping-custom.properties`
+ ** Perform a diff between the source version and the target version of `tenants/[TENANT_ID]/conf/resources-permissions-mapping.properties` and put the additional lines into the file `tenants/[TENANT_ID]/conf/resources-permissions-mapping-custom.properties`
+ ** Perform a diff between the source version and the target versionof `tenants/[TENANT_ID]/conf/dynamic-permissions-checks.properties` and put the additional lines into the file `tenants/[TENANT_ID]/conf/dynamic-permissions-checks-custom.properties`
+ ** Report all the content of the source version of``tenants/[TENANT_ID]/conf/custom-permissions-mapping.properties`` into the target version.
+24. Put a new license in the database: see xref:bonita-bpm-platform-setup.adoc#update_platform_conf[Runtime configuration] for further details.
  There is below a Linux example:
-+
+
 [source,bash]
 ----
  cd setup
  vi database.properties
  ./setup.sh pull
  ls -l ./platform_conf/licenses/
 ----
-+
+
 If there is no valid license in the `./platform_conf/licenses/`, these 2 pages will help you to request and install a new one:
 
  ** https://documentation.bonitasoft.com/?page=licenses[Licenses]
- ** xref:bonita-bpm-platform-setup.adoc#update_platform_conf[Platform configuration]
+ ** xref:bonita-bpm-platform-setup.adoc#update_platform_conf[Setup tool]
 
-+
 Install the new license:
-+
+
 [source,bash]
 ----
  cp BonitaSubscription-7.n-Jerome-myHosname-20171023-20180122.lic ./platform_conf/licenses/
  ./setup.sh push
 ----
 
-. Start the application server. Before you start Bonita Portal, clear your browser cache. If you do not clear the cache, you might see old, cached versions of Portal pages instead of the new version.
- Log in to the Portal and verify that the migration has completed.
- If you did not set the default Look & Feel before migration and you cannot log in, you need to  xref:managing-look-feel.adoc[restore the default Look & Feel] using a REST client or the Engine API.
-. *If you migrated pasted version 7.7*
- In that case, if you used the migration tool 2.41.1 or greater, the table `arch_contract_data` is automatically backed up to the table `arch_contract_data_backup` to avoid long lasting migration.
- To reintegrate the data into your installation, a new tool is provided in versions 2.46.0 and above. It is located in the `tools/live-migration` folder.
- Follow instruction in the README.md to run this tool and re-integrate data from `arch_contract_data_backup`.
+25. Start the application server. Before you start Bonita Portal (before Bonita 2021.2) and/or Bonita Applications, clear your browser cache. If you do not clear the cache, you might see old, cached versions of Portal or Applications pages instead of the new versions.
```

Comment:
and Bonita Applications (from Bonita 2021.2),

Reply:
ok